lib/statsd.rb

```diff
@@ -2,19 +2,40 @@
 require 'resolv'
 require 'forwardable'
 
-module Statsd
-  # initialize singleton instance in an initializer
+class Statsd
```

Comment:
See the commit message :-) Basically to allow us to simultaneously load this gem and another gem that declares a Statsd class -- ruby will monkeypatch them together, with minimal disruption to classes that depend on this. However, as discussed elsewhere, that may be overly fancy -- maybe we should rename the file to lookout/statsd.rb and put this in the Lookout namespace. It impacts all users, but it's cleaner and safer.